campari/AllASPFuncs.py

```diff
@@ -566,7 +566,7 @@ def constructImages(exposures, ra, dec, size=7, subtract_background=True,
     for indx, i in enumerate(exposures):
         Lager.debug(f"Constructing image {indx} of {len(exposures)}")
         band = i["BAND"]
-        pointing = i["Pointing"]
+        pointing = i["pointing"]
         sca = i["SCA"]
```

Comment:
Oh, I'd really prefer 'exp' or 'exposure" to the variable 'i' here.  In a for loop, 'i' would be assumed to be the index, not the value.

(Feel free to ignore, this is out of scope for the PR).